examples/pxScene2d/src/pxContextGL.cpp

```diff
@@ -1785,7 +1785,7 @@ class textureBorderShaderProgram: public shaderProgram
       use();
       currentGLProgram = PROGRAM_TEXTURE_BORDER_SHADER;
     }
-    glUniform2f(mResolutionLoc, resW, resH);
+    glUniform2f(mResolutionLoc, static_cast<float>(resW), static_cast<float>(resH));
```

Comment:
Let's unify when casting for uniforms.  Let's use `GLfloat` instead of a mix of `float` and `GLfloat`.